kubernetes/conf.yaml.example

```diff
@@ -11,6 +11,11 @@ instances:
   # reach the kubelet and cadvisor APIs unless the environment variable
   # KUBERNETES_KUBELET_HOST is found (if your node names can be resolved by pods, we recommend you set this variable to spec.nodeName through the downward API).
   #
+  # To enable Kubernetes State Metrics, please refer to kubernetes_states integration.
+  # For more information,
+  # please consult http://docs.datadoghq.com/integrations/kubernetes/#kubernetes-state-metrics
+  # and https://github.com/DataDog/integrations-core/ on the kubernetes_state directory.
```

Comment:
Why not just link to https://github.com/DataDog/integrations-core/tree/master/kubernetes_state?